scripts/mutect2_wdl/unsupported/funcotator.wdl

```diff
@@ -0,0 +1,153 @@
+#     annotation_overrides       -  Override values for annotations (in the format <ANNOTATION>:<VALUE>).  Replaces existing annotations of the given name with given values.
+#     gatk4_jar_override         -  Override Jar file containing GATK 4.0.  Use this when overriding the docker JAR or when using a backend without docker.
+#
+# This WDL needs to decide whether to use the ``gatk_jar`` or ``gatk_jar_override`` for the jar location.  As of cromwell-0.24,
+# this logic *must* go into each task.  Therefore, there is a lot of duplicated code.  This allows users to specify a jar file
+# independent of what is in the docker file.  See the README.md for more info.
+#
+workflow Funcotator {
+    String gatk_docker
+    File ref_fasta
+    File ref_fasta_index
+    File ref_dict
+    File variant_vcf_to_annotate
+    String reference_version
+    String output_vcf_name
+
+    String? data_sources_tar_gz
+    String? transcript_selection_mode
+    Array[String]? transcript_selection_list
+    Array[String]? annotation_defaults
+    Array[String]? annotation_overrides
+    File? gatk4_jar_override
+
+    call MakeItFunky {
+        input:
+            ref_fasta                 = ref_fasta,
+            ref_fasta_index           = ref_fasta_index,
+            ref_dict                  = ref_dict,
+            variant_vcf_to_annotate  = variant_vcf_to_annotate,
+            reference_version         = reference_version,
+            output_vcf_name           = output_vcf_name,
+            data_sources_tar_gz       = data_sources_tar_gz,
+            transcript_selection_mode = transcript_selection_mode,
+            transcript_selection_list = transcript_selection_list,
+            annotation_defaults       = annotation_defaults,
+            annotation_overrides      = annotation_overrides,
+            gatk4_jar_override        = gatk4_jar_override,
+            gatk_docker               = gatk_docker
+    }
+
+    output {
+        File annotated_vcf_out = MakeItFunky.annotated_vcf_out
+        File annotated_vcf_out_idx = MakeItFunky.annotated_vcf_out_idx
+    }
+}
+
+
+task MakeItFunky {
+    # Inputs for this task
+    File ref_fasta
+    File ref_fasta_index
+    File ref_dict
+    File variant_vcf_to_annotate
+    String reference_version
+    String output_vcf_name
+
+    File? data_sources_tar_gz
+    String? transcript_selection_mode
+    Array[String]? transcript_selection_list
+    Array[String]? annotation_defaults
+    Array[String]? annotation_overrides
+
+    # ==============
+    # Process input args:
+    String transcript_selection_arg = if defined(transcript_selection_list) then "--transcript-list " else ""
+    String annotation_def_arg = if defined(annotation_defaults) then "--annotation-default " else ""
```

Comment:
Put spaces before the `--` in all three of these.

Reply:
Fixed!